modules/visual_script/visual_script_property_selector.cpp

```diff
@@ -414,11 +419,16 @@ void VisualScriptPropertySelector::get_visual_node_names(const String &root_filt
 			String basic_type = Variant::get_type_name(vnode_function_call->get_basic_type());
 			type_name = basic_type.capitalize() + " ";
 		}
-		VisualScriptBuiltinFunc *vnode_builtin_function_call = Object::cast_to<VisualScriptBuiltinFunc>(*VisualScriptLanguage::singleton->create_node_from_name(E->get()));
-		if (vnode_builtin_function_call != NULL) {
-			type_name = "Builtin ";
+
+		Vector<String> desc = path[path.size() - 1].replace("(", "( ").replace(")", " )").replace(",", ", ").split(" ");
```

Comment:
I agree we're not dealing with huge portions of data, but I think this is adding more complexity. Before this change, the space was included directly with the parenthesis. 

If you replace every instance of `"("` by `"( "`, why not doing this in value assignation as it was done before ?

Reply:
The problem I ran into was for some reason String::capitalize was formatting strings of the form:

blah/blah/function_name(arg1,arg2) to blah/blah/function_name( arg1, arg2) due to complexity of the above Boolean previously mentioned.

Reply:
Another way of saying that is: "( " -> "(" was being ignored by the capitalize function.

Reply:
Indeed, I am surprized that the `capitalize()` function doesn't ignore non alphabetic chars. I am not certain this is the desired behaviour, so maybe it requires a fix @bojidar-bg ?

Reply:
`capitalize` is a bit wierd, it turns "something/function_name(arg1,arg2)" into "Something/function Name(arg 1,arg 2)" actually.

Not sure what can be done for it though.

Reply:
Fine, then I guess it's ok as it is.

Reply:
I added some test cases.

Reply:
Any idea how to resolve these?  I know we decided to not change these, but it's annoying me.

@bojidar-bg 

```
        input = "function(name)";
	output = "Function(name)";
 	input = "snake_case_function(snake_case_arg)";
	output = "Snake Case Function(snake Case Arg)";
```
`

Reply:
I think the `(` should be treated as a space, so `output_2="Snake Case Function(Snake Case Arg)"`